packages/cms-data-sync/src/utils/setup.ts

```diff
@@ -52,5 +54,13 @@ export const getSquidexAndContentfulClients = async () => {
     CONTENTFUL_ENV_ID!,
   );
 
+  if (disableWebhook) {
```

Comment:
have you thought about how to enable the webhook back? Once the migration is done it needs to be returned to it's normal state.

the issue with making this part of the client is that we only get the client once, so I think you may want to move this into the migration script, somewhere at the beginning
https://github.com/yldio/asap-hub/blob/40b0c652abaaff7975822d2196795ed4573886bb/packages/cms-data-sync/src/index.ts#L6
you might also want to make sure that should something go wrong during the migration, the webhooks are still enabled back